solarforecastarbiter/cli.py

```diff
@@ -52,8 +52,13 @@ def cli_access_token(user, password):
     try:
         token = request_cli_access_token(user, password)
     except requests.HTTPError as e:
-        click.echo(click.style(
-            e.response.json()['error_description'], fg='red'))
+        try:
+            message = e.response.json()['error_description']
+        except json.decoder.JSONDecodeError:
+            message = e.response.text
+            if not message and e.response.status_code == 500:
+                message = 'Internal Server Error'
+        click.echo(click.style(message, fg='red'))
```

Comment:
When trying to understand the unhandled JSONDecodeError exception, I accidentally committed a print statement here. I figured I might as well clean up the exception when removing the print statement, but didn't put much thought into it. I can revert if that's better.

```solararbiter referencedata update --network 'PNNL' --base-url 'https://dev-api.solarforecastarbiter.org' --user "testing@solarforecastarbiter.org" --password=Thepassword123! --start 20171231 --end 20180102 -vvv
2021-04-20 12:52:37,364 DEBUG Starting new HTTPS connection (1): solarforecastarbiter.auth0.com:443
2021-04-20 12:52:37,604 DEBUG https://solarforecastarbiter.auth0.com:443 "POST /oauth/token HTTP/1.1" 500 0
Internal Server Error
```